unstructured_inference/models/yolox.py

```diff
@@ -66,6 +87,7 @@ def initialize(self, model_path: str, label_map: dict):
                 "CPUExecutionProvider",
             ],
         )
+
         self.layout_classes = label_map
 
     def image_processing(
```

Comment:
btw, given the doc string this function should likely be called `yolox_` something (image_to_layout?).

Reply:
Probably. We don't have a convention for that kind of methods. Detectron2 use `predict`, wouldn't harm to change the naming in another PR